SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -247,16 +285,29 @@ bool USpatialLatencyTracer::BeginLatencyTrace_Internal(const AActor* Actor, cons
 
 	WriteKeyFrameToTrace(&NewTrace, FString::Printf(TEXT("Begin trace : %s"), *FunctionName));
 
+	// For non-spatial tracing
+	const improbable::trace::SpanContext& TraceContext = NewTrace.context();
+
+	OutLatencyPayload.TraceId = TArray<uint8_t>((const uint8_t*)&TraceContext.trace_id()[0], sizeof(improbable::trace::TraceId));
+	OutLatencyPayload.SpanId = TArray<uint8_t>((const uint8_t*)&TraceContext.span_id()[0], sizeof(improbable::trace::SpanId));
+
 	TraceMap.Add(Key, MoveTemp(NewTrace));
 
+	if (!GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking())
```

Comment:
Nit: This logic is written in three places. Could pull out a function called something along the lines of HandleTraceMapReset or maybe OnTraceMapUpdateEnd. 

Again not really necessary.

Reply:
Addressed these comments!